test/mocha/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I recommend we use `.eslintrc.js` and do `module.exports = ` here (with semi-colon at the end). That way we keep things consistent ... always using the `.js` extension / JS version of the lint config file.

Reply:
Updated e1d4fb1